README.md

```diff
@@ -12,45 +12,58 @@ This open-source project is developed, maintained and supported by [UCF consorti
 ## Installation
 
 ### Obtain SparkUCX
-Please use the ["Releases"](https://github.com/openucx/sparkucx/releases) page to download pre-built binaries.
+Please use the ["Releases"](https://github.com/openucx/sparkucx/releases) page to download SparkUCX jar file
```

Comment:
need section with quick start on cluster with mofed installed, i.e.
```
% wget sparkucx.tar.gz
% tar zxvf sparkucx.tar.gz
% export CLUSTER_SHARED_DIR=/shared/local/sparkucx
% export UCX_PREFIX=/opt/mellanox/ucx
% cp spakucx.jar $CLUSTER_SHARED_LOCATION/lib
% export

Reply:
In [another PR](https://github.com/openucx/sparkucx/pull/3/files#diff-7856a8cfc33697fb151432cd13d3fa6a) - there's a script for github action. 

Seems ike there's [a tool](https://github.com/systemslab/popper) to convert github action script to other platforms:
```
Other Runtimes. In addition to Docker, Popper can execute workflows in other container runtimes. We currently support Singularity and are working on adding Podman and Vagrant to the list (see here for more).
```
Will check it.

Reply:
Also, SparkUCX requires ucx-1.7 that's not yet released - hence not in MOFED. So for now need to build ucx manually.

Reply:
i mean, create container with distro,ofed,ucx,sparkucx,spark and offer it as fastpath to run workload

Reply:
+1 on containers. IMHO very good idea. We should prepare one for UCX with all the add-ons like CUDA, KNEM , etc.

Reply:
@petro-rudenko - lets handle container as separate task, not in the context of this PR.